### PR TITLE
Added Styling for Readonly View of Forms

### PR DESCRIPTION
**Pull request by ruben-bottu**
image

### CODE REVIEW

File: app/components/form/instance.hbs [hunk 1]
```diff
@@ -15,7 +15,7 @@
             @forceShowErrors={{true}}
             @useNewListingLayout={{true}}
             @level={{2}}
-            class="au-u-max-width-medium"
+            class="au-u-max-width-medium {{if this.isEditable '' 'disabled'}}"
```
Comment (bfidlers) [inline app/components/form/instance.hbs:18]: I find this ternary if here a bit confusing and it doesn't seem necessary, I would suggest using 
`{{unless this.isEditable 'disabled'}}`

File: app/templates/organen/orgaan.hbs [hunk 2]
```diff
@@ -20,15 +20,20 @@
...
+  />
+
+  <div class="au-o-layout">
+    <AuButton
+      class="au-u-margin-bottom"
+      @skin="secondary"
+      {{on "click" this.edit}}
+    >Pas aan</AuButton>
 
-<AuToolbar @border="top" @size="large" as |Group|>
```
Comment (bfidlers) [inline app/templates/organen/orgaan.hbs:29]: Why did you remove the toolbar here, seems like a good practice to use those?

File: app/styles/_shame.scss
Comment (bfidlers) [inline app/styles/_shame.scss:1]: We will probably need to review these rules when we use disabled forms with other fields, but looks good for now.

**Review by bfidlers — APPROVED** [review]
Few comments, but otherwise looks okay.

File: app/components/form/instance.hbs (hunk 1 above)
Comment (ruben-bottu) [reply app/components/form/instance.hbs:18]: Oh didn't know you could do that, thanks!

File: app/templates/organen/orgaan.hbs (hunk 2 above)
Comment (ruben-bottu) [reply app/templates/organen/orgaan.hbs:29]: Because we wanted to make the "Pas aan" look inline with the form. It might be strange to have it in the toolbar as well no?

Comment (ruben-bottu) [reply app/templates/organen/orgaan.hbs:29]: And karel wanted the todo to be directly under the form

File: app/styles/_shame.scss [hunk 3]
```diff
@@ -127,3 +127,31 @@
...
+  background-color: transparent;
+  color: black;
+  border: transparent;
+  padding-left: 0;
+}
+
+.disabled .ember-power-select-trigger[aria-disabled="true"] {
+  background-color: transparent;
+  border: transparent;
+  padding-left: 0 !important;
```
Comment (ruben-bottu) [inline app/styles/_shame.scss:144]: This !important is sadly necessary, because it is also !important in ember-submission-form-fields

**Review by bfidlers — APPROVED** [review]
